src/main/java/gov/usgs/wma/waterdata/LoadDiscreteGroundWater.java

```diff
@@ -60,7 +61,11 @@ public int loadDiscreteGroundWaterIntoObservationDb (List<DiscreteGroundWater> d
 		if (count == discreteGroundWaterList.size()) {
 			LOG.debug(String.format(INSERT_SUCCEEDED_MESSAGE, fieldVisitIdentifier));
 		} else {
-			String failMessageInsertFailed = String.format(INSERT_FAILED_MESSAGE, discreteGroundWaterList.size(), count, fieldVisitIdentifier);
+			String failMessageInsertFailed = String.format(
+					INSERT_FAILED_MESSAGE,
+					discreteGroundWaterList.size(),
+					count, fieldVisitIdentifier,
+					discreteGroundWaterList.get(0).getMonitoringLocationIdentifier());
```

Comment:
The monitoring location comes from information we get from the header info on an inspection activity, so it will be the same no matter how many gw levels are associated with this field_visit_identifier, so I felt ok about using `discreteGroundWaterList.get(0)` here.